appendix-keyboard.tex

```diff
@@ -130,7 +130,7 @@ \section{Synthetic Key Events}
 The MEGA65 keyboard interface logic allows the use of a variety of keyboard types and alternatives. This is partly
 to cater for the early development on general purpose FPGA boards, the MEGAphone with its touch interface, and the
 desktop versions of the MEGA65 architecture.  In addition to these real keyboard types, the MEGA65 also allows the
-injecting of synthetic key events via the registers \$D615 -- \$D617 (decimal 54,805 -- 54,807).  By setting the
+injecting of up to 3 concurrent key events via the registers \$D615 -- \$D617 (decimal 54,805 -- 54,807).  By setting the

```

Comment:
On reflection, here what we want to communicate is that "The depressing of up to 3 three keys can be simulated via the registers ..."  This makes it clearer that what happens is not that a single key event occurs, but it is as though that key is being held down for as long its scan-code is in one of these registers.

Reply:
Fixed 4849a1417de9cb0a849e49da6eb3ad5658e53b5d